apertium/transfer.h

```diff
@@ -61,7 +61,14 @@ class Transfer
   Buffer<TransferToken> input_buffer;
   vector<wstring *> tmpword;
   vector<wstring *> tmpblank;
-
+  
+  //for secondary tags
+  bool in_out_lu; //flag to denote that lu in out is being processed
+  string stags; //stores secondary tags of the LU that is being output
```

Comment:
I know this is very "taste-based" (and languages/frameworks have their own conventions) but I personally prefer slightly longer variable names that are more meaningful. So if instead of `stags` it was `secondary_tags`, it would be much easier to follow the code.

Reply:
I agree. Will make the change :)

Reply:
What about in the t1x? secondary_tags will be too long. stags should be fine for that right? maybe sec_tags if needed.